docs/user/integration/aws-cloudwatch/README.md

```diff
@@ -0,0 +1,175 @@
+1. Enter the user name and click **Next**.
+1. Select **Attach policies directly**.
+1. Select the two policies you created previously, as well as the policy **AWSXrayWriteOnlyAccess**.
+1. Click **Next** > **Create User**.
+1. Open the new user.
+1. Under **Security credentials**, click **Create access key**.
+1. Select **Application running outside AWS** and click **Next**.
+1. Describe the purpose of this access key and click **Create access key**.
+1. Copy and save the access key and the secret access key.
+
+### Create a Secret with AWS Credentials
+
+To connect the AWS Distro to the AWS services, create a secret containing the credentials of the created IAM user into the Kyma cluster. In the following command, replace `{ACCESS_KEY}` with your access key, `{SECRET_ACCESS_KEY}` with your secret access key, and `{AWS_REGION}` with the AWS region you want to use:
+ 
+```bash
+kubectl create secret generic aws-credentials -n $K8S_NAMESPACE --from-literal=AWS_ACCESS_KEY_ID={ACCESS_KEY} --from-literal=AWS_SECRET_ACCESS_KEY={SECRET_ACCESS_KEY} --from-literal=AWS_REGION={AWS_REGION}
+ ```
+
+## Deploy the AWS Distro
+
+Deploy the AWS Distro which is a specific distribution of an OTel collector. It converts and dispatches the OTLP-based metrics and traces in the cluster to the AWS-specific format and protocol.
+
+ ```bash
+kubectl -n $K8S_NAMESPACE apply -f aws-otel.yaml
+ ```
+
+## Set Up Kyma Pipelines
+
+Use the Kyma Telemetry module to enable ingestion of the signals from your workloads:
+
+1. Deploy a LogPipeline:
+   ```bash
+   kubectl apply -f - <<EOF
+   apiVersion: telemetry.kyma-project.io/v1alpha1
+   kind: LogPipeline
+   metadata:
+     name: aws-cloudwatch
+   spec:
+     output:
+       custom: |
+         Name cloudwatch_logs
+         region \${AWS_REGION}
+         auto_create_group On
+         log_group_template /logs/\$cluster_identifier
+         log_group_name /logs/kyma-cluster         
+         log_stream_template \$kubernetes['namespace_name'].\$kubernetes['pod_name'].\$kubernetes['container_name']
+         log_stream_name from-kyma-cluster
+         log_retention_days 7
+     variables:
+       - name: AWS_ACCESS_KEY_ID
+         valueFrom:
+           secretKeyRef:
+             name: aws-credentials
+             namespace: $K8S_NAMESPACE
+             key: AWS_ACCESS_KEY_ID
+       - name: AWS_SECRET_ACCESS_KEY
+         valueFrom:
+           secretKeyRef:
+             name: aws-credentials
+             namespace: $K8S_NAMESPACE
+             key: AWS_SECRET_ACCESS_KEY
+       - name: AWS_REGION
+         valueFrom:
+           secretKeyRef:
+             name: aws-credentials
+             namespace: $K8S_NAMESPACE
+             key: AWS_REGION
+   EOF
+   ```
+   > NOTE: For now, the logs integration is inconsistent with the metrics and traces integration. It will be updated to the common approach as soon as the AWS Distro supports logs and the Kyma Telemetry module uses OTLP logs.
```

Comment:
do we really need this here? Will you remember to throw out this Note when after logging switched to OTLP? I'd recommend to drop this Note.

Reply:
The reason is not correct anymore. Maybe remove this note and add a sentence to the first paragraph, where you mention the AWS Distro that we ingest logs directly from Fluent Bit to CloudWatch and have this detour for metrics and traces.